entries/a/airasia.com.json

```diff
@@ -0,0 +1,13 @@
+{
+  "AirAsia": {
+    "domain": "airasia.com",
+    "tfa": [
+      "email"
+    ],
+    "categories": [
+      "retail",
+      "transport",
+      "travel"
+    ]
```

Comment:
I'm guessing they have one account for all of [Capital A's brands](https://www.capitala.com/our_brands.html), specifically AirAsia and AirAsia MOVE?

Reply:
AirAsia MOVE seems to just be the branding for their digital platforms.